pkg/clusteragent/autoscaling/workload/controller_test.go

```diff
@@ -45,6 +45,8 @@ const testMaxAutoscalerObjects int = 2
 
 func newFixture(t *testing.T, testTime time.Time) *fixture {
 	store := autoscaling.NewStore[model.PodAutoscalerInternal]()
+	InitDumper(store)
```

Comment:
Why do we need it? Except in `dump_test.go` I don't expect us to use the variable?

Reply:
`dump_test.go` is currently reusing this code to generate the fixture, but I can create a new test helper function to keep them separate

Reply:
You can use the fixture and call `InitDumper()` in your test